synapse/handlers/room_summary.py

```diff
@@ -452,7 +452,6 @@ async def _summarize_local_room(
                 "type": e.type,
                 "state_key": e.state_key,
                 "content": e.content,
-                "room_id": e.room_id,
```

Comment:
Just to confirm: Would `room_id` here be the room ID of the space?

Reply:
> Just to confirm: Would `room_id` here be the room ID of the space rather than the child?

Correct, the `state_key` is what contains the room ID of the child. The original API design didn't have the children nested under the space, so you had to have to room ID in each child to connect it back to the parent. This was dropped as part of the MSC process, but we never dropped it from Synapse (an oversight)...